Tests/Pcap++Test/Tests/FilterTests.cpp

```diff
@@ -668,6 +667,29 @@ PTF_TEST_CASE(TestPcapFiltersOffline)
 	filterVec.push_back(&protoFilter);
 	pcpp::AndFilter andFilter(filterVec);
 	andFilter.parseToString(filterAsString);
+	PTF_ASSERT_EQUAL(filterAsString, "(ip and src net 10.0.0.6) and (udp)");
+
+	andFilter.addFilter(&ipFilter);
+	andFilter.parseToString(filterAsString);
+	PTF_ASSERT_EQUAL(filterAsString, "(ip and src net 10.0.0.6) and (udp) and (ip and src net 10.0.0.6)");
+
+	andFilter.removeFilter(&ipFilter);
+	andFilter.parseToString(filterAsString);
+	PTF_ASSERT_EQUAL(filterAsString, "(udp) and (ip and src net 10.0.0.6)");
+
+	{
+		pcpp::OrFilter externalFilter;
+		andFilter.removeFilter(&externalFilter);
+		PTF_ASSERT_EQUAL(filterAsString, "(udp) and (ip and src net 10.0.0.6)");
+	}
+
+	andFilter.clearAllFilters();
+	andFilter.parseToString(filterAsString);
+	PTF_ASSERT_EQUAL(filterAsString, "");
+
+	andFilter.setFilters(filterVec);
```

Comment:
this seems redundant?  line 668 already does the same thing.

Reply:
668 tests that the vector ctor works correctly. This one tests `setFilters` specifically.

Reply:
ops, sorry, I didn't notice that.